example/proxy_conf.yaml

```diff
@@ -109,7 +110,7 @@ LOGGING:
     backends.spidsaml2:
       level: INFO
       formatter: simple
-      handlers: [spid_daily]
```

Comment:
with spid daily we apply the SPID retention policy, we can enable multiple handlers and I suggest to keep the spid_daily up and running as previously configured

Reply:
is there any update for this log handler?
may we leave it as commented, to let the user decide to collect the spid related logs in a separate log file?